Wikipedia/Code/SavedViewController.swift

```diff
@@ -225,7 +275,50 @@ class SavedViewController: ThemeableViewController, WMFNavigationBarConfiguring,
             }
         }
 
-        configureNavigationBar(titleConfig: titleConfig, closeButtonConfig: nil, profileButtonConfig: nil, searchBarConfig: searchConfig, hideNavigationBarOnScroll: hidesNavigationBarOnScroll)
+        configureNavigationBar(titleConfig: titleConfig, closeButtonConfig: nil, profileButtonConfig: profileButtonConfig, searchBarConfig: searchConfig, hideNavigationBarOnScroll: hidesNavigationBarOnScroll)
+    }
+    
+    @objc func userDidTapProfile() {
+        
+        guard let dataStore else {
+            return
+        }
+        
+        guard let languageCode = dataStore.languageLinkController.appLanguage?.languageCode,
+              let metricsID = DonateCoordinator.metricsID(for: .savedProfile, languageCode: languageCode) else {
+            return
+        }
+        
+        // TODO: Do we need logging like this?
+        // DonateFunnel.shared.logExploreProfile(metricsID: metricsID)
```

Comment:
I asked Shay in the data sync yesterday, she said we only log it if we're trying to validate something specific. In this case, we might not log it.